shared/billing/__init__.py

```diff
@@ -37,6 +37,7 @@ def is_enterprise_cloud_plan(plan: BillingPlan) -> bool:
 
 
 def is_pr_billing_plan(plan: str) -> bool:
+    # use is_pr_billing_plan() in PlanService instead of accessing this directly
```

Comment:
Maybe it's worth to mark it as [deprecated](https://typing.readthedocs.io/en/latest/spec/directives.html#deprecated)?